src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx

```diff
@@ -74,7 +74,7 @@ export const DemoCreateIssue: FunctionComponent = () => {
                 </a>
                 .
               </p>
-              <Link to="/demo/verify">
+              <Link href="/demo/verify">
```

Comment:
hum button is not a link? according to the doc if it's a link must use passRef and eventually forward ref

Also a11y wise, button should be alink